src/c++/perf_analyzer/inference_profiler.h

```diff
@@ -440,17 +446,22 @@ class InferenceProfiler {
   /// sequence model.
   /// \param latencies Returns the vector of request latencies where the
   /// requests are completed within the measurement window.
+  /// \param response_latencies Returns the vector of response latencies when
```

Comment:
Is this comment true? Is nothing returned here for coupled models?

Reply:
it would be empty yes.